test/batchReceiver.spec.ts

```diff
@@ -213,4 +213,157 @@ describe("ReceiveBatch from Queue/Subscription", function(): void {
 
     await receivedMsgs[0].complete();
   });
+
+  it("Abandoning a message 10 times, you get it no more. You can then only get it from the dead letter queue", async function(): Promise<
+    void
+  > {
+    await queueClient.send(testMessages[0]);
+    let abandonMsgCount = 0;
+    let receivedMsgs;
+
+    while (abandonMsgCount < 10) {
```

Comment:
Let's have this 10 as a constant (`maxDeliveryCount`) and define it outside in the describe() method.